### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by ovflowd**

### CODE REVIEW

File: locale/en/docs/guides/blocking-vs-non-blocking.md [hunk 1]
```diff
@@ -39,15 +39,15 @@ execute **asynchronously**.
 Using the File System module as an example, this is a **synchronous** file read:
 
 ```js
-const fs = require('fs');
-const data = fs.readFileSync('/file.md'); // blocks here until file is read
+const fs = require("fs");
```
Comment (mhdawson) [inline locale/en/docs/guides/blocking-vs-non-blocking.md:42]: I think I most often see ''' used versus '"' so this change seems odd to me and unrelated to the translation?

Comment (nschonni) [reply locale/en/docs/guides/blocking-vs-non-blocking.md:42]: It was an unrelated change in https://github.com/nodejs/nodejs.org/pull/4999

**Comment by ovflowd** [comment]
@mhdawson just a note that Crowdin is using my GitHub account to post the translations 😅 I didn't create this PR... @Trott we might need to re-setup the integration with a bot account.

**Comment by Trott** [comment]
> we might need to re-setup the integration with a bot account

The GitHub bot is already an admin in Crowdin so it should hopefully be pretty easy to make that happen.

**Comment by arzanilorestan** [comment]
The best time is popular when it's gone. If you want to come to your reputation and power, you must stay a secret.

**Comment by ovflowd** [comment]
@Trott we need to resetup the integration with the GitHub account of the bot, then. I can show you how to do and you can do it if you want :)

**Comment by Trott** [comment]
> @Trott we need to resetup the integration with the GitHub account of the bot, then. I can show you how to do and you can do it if you want :)

Is it a matter of doing https://support.crowdin.com/github-integration/ while logged in as the bot? I don't have the bot credentials, but presumably someone on @nodejs/build does.....

**Comment by ovflowd** [comment]
Exactly! Just need to be logged in as the bot.

**Comment by Trott** [comment]
Created and added a build-agenda label so this gets added to the next Build WG agenda, but hopefully it can be resolved before then (and apologies to @nodejs/build if this isn't a Build issue, but I'm guessing it is).

**Comment by richardlau** [comment]
> Created and added a build-agenda label so this gets added to the next Build WG agenda, but hopefully it can be resolved before then (and apologies to @nodejs/build if this isn't a Build issue, but I'm guessing it is).

This is a Build WG issue, although to set expectations, while Build does own the bot, most of @nodejs/build-infra ([including myself](https://github.com/nodejs/github-bot/issues/335)) do not currently have access to the appropriate folder in build's secrets repository which includes the account information[1].

[1]: There's a `github_bot_account.md` file but I can't decrypt the contents.

**Comment by ovflowd** [comment]
cc @mhdawson maybe you have access to the file @richardlau is mentioning?

**Comment by mhdawson** [comment]
@ovflowd we are already discussing how we get the credentials and make them accessible to the build wg, but @richardlau was setting expectations that things don't move fast and there currently is a blocking issue.

**Comment by ovflowd** [comment]
Closing for now, and let's re-run the Crowdin sync after #4991

**Pull request closed without merging**